vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -360,8 +363,21 @@ func (d *VolumeDriver) MountVolume(name string, fstype string, id string, isRead
 	return mountpoint, fs.MountWithID(mountpoint, fstype, id, isReadOnly)
 }
 
+//VolumesInRefMap - get list of volumes from refmap
```

Comment:
another nit - please mention in the comments that these are full names <volume@datastore> format